packages/mask/src/components/shared/WalletStatusBox/index.tsx

```diff
@@ -3,107 +3,127 @@ import {
+                (isDashboardPage() ? getMaskColor(theme).primaryBackground2 : theme.palette.background.default),
+            borderRadius: 8,
+            alignItems: 'center',
+        },
+        dashboardBackground: {
+            background: theme.palette.background.default,
+        },
+        accountInfo: {
+            fontSize: 16,
+            flexGrow: 1,
+            marginLeft: theme.spacing(1.5),
+        },
+        accountName: {
+            color: theme.palette.maskColor.dark,
+            fontWeight: 700,
+            fontSize: 14,
+            marginRight: 5,
+            lineHeight: '18px',
+        },
+        balance: {
+            color: theme.palette.maskColor.dark,
+            fontSize: 14,
+            paddingTop: 2,
+            lineHeight: '18px',
+        },
+        infoRow: {
+            display: 'flex',
+            alignItems: 'center',
+        },
+        actionButton: {
+            fontSize: 12,
+            color: 'white',
+            backgroundColor: theme.palette.maskColor.dark,
+            marginLeft: theme.spacing(1),
+            padding: theme.spacing(1, 2),
+            '&:hover': {
+                backgroundColor: theme.palette.maskColor.dark,
+            },
+        },
+        address: {
+            fontSize: 16,
+            marginRight: theme.spacing(1),
+            display: 'inline-block',
+        },
+        link: {
+            color: theme.palette.maskColor.dark,
+            fontSize: 14,
+            display: 'flex',
+            alignItems: 'center',
+        },
+        twitterProviderBorder: {
+            width: 14,
+            height: 14,
+        },
+        connectButtonWrapper: {
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            margin: theme.spacing(2, 0),
+        },
+        icon: {
+            fill: 'none',
+            width: 17.5,
+            height: 17.5,
+            marginRight: theme.spacing(0.5),
+        },
+        statusBox: {
+            position: 'relative',
+        },
+    }),
+)
 interface WalletStatusBox {
     disableChange?: boolean
     showPendingTransaction?: boolean
 }
 export function WalletStatusBox(props: WalletStatusBox) {
     const { t } = useI18N()
 
-    const { classes } = useStyles()
+    const providerDescriptor = useProviderDescriptor<'all'>()
+    const { classes } = useStyles({
+        contentBackground: providerDescriptor?.backgroundGradient,
+        showWalletName: [ProviderType.MaskWallet, 'Phantom', 'Blocto'].includes(providerDescriptor?.type ?? ''),
+    })
+    const connection = useWeb3Connection(NetworkPluginID.PLUGIN_EVM)
     const chainId = useChainId()
     const account = useAccount()
     const wallet = useWallet()
-    const providerType = useProviderType()
-    const providerDescriptor = useProviderDescriptor()
+    const { value: balance = '0' } = useBalance()
+    const { value: nativeToken } = useNativeToken(NetworkPluginID.PLUGIN_EVM)
```

Comment:
It cannot suppose the EVM chain is used.

Reply:
not implemented yet.